app/views/components/show.html.erb

```diff
@@ -5,15 +5,20 @@
 <%= link_to 'Upload', new_component_certificate_path(@component), class: 'govuk-button' %>
 
 <h3 class="govuk-heading-m">Signing Certificates (Enabled)</h3>
- <%= render "certificates/certificates_table", 
+ <%= render "certificates/certificates_table",
      certificates: @component.enabled_signing_certificates
  %>
 
 <h3 class="govuk-heading-m">Signing Certificates (Disabled)</h3>
- <%= render "certificates/certificates_table", 
+ <%= render "certificates/certificates_table",
      certificates: @component.disabled_signing_certificates
  %>
 
+ <h3 class="govuk-heading-m">Encryption Certificate</h3>
+  <%= render "certificates/certificates_table",
+     certificates: @component.encryption_certificate ? [@component.encryption_certificate] : []
```

Comment:
I would move this condition outside of the `render` ... there's no point in rendering the partial if we're not passing anything in